Lib/test/test_winreg.py

```diff
@@ -1,6 +1,7 @@
 # Test the windows specific win32reg module.
 # Only win32reg functions not hit here: FlushKey, LoadKey and SaveKey
 
+import contextlib
```

Comment:
Did you leave this import here for a reason?

Reply:
Nope, accident. Thanks!